lib/situation_room/sites.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Situation_Room.Sites do
```

Comment:
In Elixir, we typically name the models for schema objects in the Singluar.  So this should be `SituationRoom.Site`